myapp/app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container mt-5">
+  <h1 class="mb-4">Tasks</h1>
+
+  <%= link_to 'New Task', new_task_path, class: 'btn btn-primary mb-3' %>
+
+  <div class="row">
+    <% @tasks.each do |task| %>
+      <div class="col-md-3"> <!-- 调整列的宽度 -->
+        <div class="card mb-4" style="height: 250px;"> <!-- 调整卡片的高度 -->
+          <div class="card-body d-flex flex-column">
+            <h5 class="card-title">Title: <%= link_to task.title, task %></h5> <!-- 添加链接到 show 页面 -->
+            <p class="card-text" style="flex-grow: 1; max-height: 100px; overflow: hidden;">Description: <%= task.description %></p>
```

Comment:
I think you can use it.
https://getbootstrap.jp/docs/5.3/utilities/flex/#%e6%8b%a1%e5%a4%a7%e3%81%a8%e7%b8%ae%e5%b0%8f
```suggestion
            <p class="card-text flex-grow-1" style="max-height: 100px; overflow: hidden;">Description: <%= task.description %></p>
```

Reply:
Thank you！ I changed it
、